tests/v0_tests/test_add_documents.py

```diff
@@ -618,4 +620,111 @@ def test_multimodal_combination_doc(self):
             self.warm_request(self.client.index(self.index_name_1).search,"")
 
         space_tensor_res = self.client.index(self.index_name_1).search("")
-        assert space_tensor_res["hits"][0]["_id"] == "111"
\ No newline at end of file
+        assert space_tensor_res["hits"][0]["_id"] == "111"
+
```

Comment:
These little unit tests are awesome! It really helps us be certain about the behaviour of py-marqo going forward!